internal/types/document.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+const (
+	// IdKey is the key for _id field.
+	IdKey = "_id"
```

Comment:
Actually, do we need this const to be exported? Or can we leave it unexported?

```suggestion
	idKey = "_id"
```

Reply:
We probably don't need it, unexported it.